apps/scan/frontend/src/components/layout.tsx

```diff
@@ -21,33 +25,85 @@ const ELECTION_BAR_HIDDEN_SIZE_MODES: ReadonlySet<SizeMode> = new Set([
 ]);
 
 export interface ScreenProps {
+  actionButtons?: React.ReactNode;
   ballotCountOverride?: number;
   centerContent?: boolean;
   children: React.ReactNode;
   isLiveMode?: boolean;
   infoBarMode?: InfoBarMode;
   padded?: boolean;
-  title?: string;
+  title?: React.ReactNode;
+  voterFacing: boolean;
 }
 
 export type CenteredScreenProps = Omit<ScreenProps, 'centered' | 'padded'>;
 
+const CONTENT_SPACING_VALUES_REM: Readonly<Record<SizeMode, number>> = {
+  desktop: 0.5, // unused
```

Comment:
Just for my understanding, why exactly is this not used? Does the desktop layout not use these components?

Reply:
That's right - these components are only used with the `touch*` size modes, so this is really just for type completeness